config/routes/admin.rb

```diff
@@ -67,6 +67,9 @@
     end
 
     resources :gdpr_delete_requests, only: %i[index destroy]
+
+    resources :bulk_assign_role, only: %i[index]
+    post "/bulk_assign_role/assign_role", to: "bulk_assign_role#assign_role"
```

Comment:
Would love to hear others thoughts here as well, but could we maybe just use the post resource for bulk_assign_role here instead of assigning this url. The path name reads a bit strangely too `admin_bulk_assign_role_assign_role_path`.

Reply:
That makes sense to me, too — :+1: on using the custom route here.

Reply:
This one I am not sure what to change.
To make things a little better I have updated the path to `admin_bulk_assign_role_path` and `post "/bulk_assign_role", to: "bulk_assign_role#assign_role"`

let me know how can improve it further.